MlabCommunity/mLAB2024-frontend
# dashboard initial view

In this pull request, I am adding a new layout containing a dashboard. Below are the details of the changes made:

Created a new Dashboard component and NavBar, SideNav
modyfication layout.tsx and page.tsx 

Added sample widgets and data to the dashboard.
How to test
Switch to the your-branch branch.
Run the application.
Navigate to the dashboard page.
Verify the layout displays correctly
